.github/workflows/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+
+  attr_reader :id, :name, :age
+  attr_writer :name, :age
+
+  def initialize (name="unknown",age,parent_permission=true)
+    @id=rand(1..1_000)
+    @name=name
+    @age=age
+    @parent_permission=parent_permission
+  end
```

Comment:
- [ ] In order to avoid errors, optional parameters should come last, so `age` should come before `name="unknown"`